### PR TITLE
Feature objects distribution

### PR DESCRIPTION
**Pull request by mixcas**
add fields for object position
loads object in scene at their positions

### CODE REVIEW

File: index.php [hunk 1]
```diff
@@ -20,12 +20,15 @@
     if( $meta["_igv_obj_file"] && $meta["_igv_mtl_file"] ) {
       echo '
     Models.push({
-      obj: "' . $meta["_igv_obj_file"][0] . '",
-      mtl: "' . $meta["_igv_mtl_file"][0] . '",
-    });
-  </script>';
+      obj: "' . $meta["_model_obj_file"][0] . '",
```
Comment (varffs) [inline index.php]: does this even work without the `_igv` prefix?!!?

Comment (varffs) [inline index.php]: the answer is no! u even test this?

Comment (varffs) [inline index.php]: :[ :-1:

File: lib/meta-boxes.php [hunk 2]
```diff
@@ -32,7 +41,7 @@ function get_post_objects( $query_args ) {
 function igv_cmb_metaboxes() {
 
 	// Start with an underscore to hide fields from custom fields list
-	$prefix = '_igv_';
+	$prefix = '_model_';
```
Comment (varffs) [inline lib/meta-boxes.php]: ah ok but why?!

**Comment by mixcas** [comment]
I hated `_igv_obj_file` and `_igv_pos_x`.
Maybe `_igv_model_` works better as prefix?

**Comment by varffs** [comment]
but you then broke every other meta field on the website! I'm fixing now

**Pull request merged**